wpilibc/src/main/native/include/frc/RobotBase.h

```diff
@@ -33,6 +31,12 @@ void RunRobot(wpi::mutex& m, Robot** robot) {
     theRobot.StartCompetition();
   } catch (const frc::RuntimeError& e) {
```

Comment:
Should we catch std::exception as a separate block too?  Or even ...?

Reply:
Since all we're doing is sending the exception text to the DS console then rethrowing, idk if there's anything we can do with `catch (...)`. What's the intended way to send an exception to the console? I see `HAL_SendError()` (really verbose), `FRC_ReportError()` (takes an error code and format string but no exception or stacktrace), and some mix of the two involving wpi/StackTrace.h in wpilibc classes.

Reply:
`frc::RuntimeError::Report()` uses `HAL_SendError()` under the hood.  So with a raw `std::exception` you could use `HAL_SendError()` with the `what()` output.  Stacktrace is specific to `frc::RuntimeError`, you can't get a stacktrace at this point (the stack has already unwound).

Reply:
It's better to use HAL_SendError than just printing so it shows up in the right level of filtering in the DS console log